.circleci/config.yml

```diff
@@ -299,8 +303,8 @@ jobs:
           name: deploy to s3
           command: 'aws s3 sync ./dist s3://$BUCKET_NAME/release'
       - run:
-          name: Trigger Windows build
-          command: './scripts/trigger_windows_build.sh ${APPVEYOR_TOKEN} ${CIRCLE_SHA1} release'
+          name: Deploy to Grafana.com
```

Comment:
When 5.3 stable is released we remove the grafana appveyor build over at appveyor right?

Reply:
Okay. Minor thing just curious - releases are not necessarily made from master, i.e. 5.3.x etc. Will appveyor build all branches as well?

Reply:
@marefr That's a good point, and no, not currently. We talked about moving the appveyor trigger to the build/build-all jobs which would solve this issue.